src_new/creative/owCreativeRenderer.js

```diff
@@ -0,0 +1,44 @@
+require("prebid-universal-creative");
+
+window.PWT = window.PWT || {};
+window.pbjs = window.pbjs || {};
+
+var refThis = this;
+
+// Desc: Uses prebid js render ad function to call cache and render ad based on the params from response.
+// Param Defination 
+exports.renderCreative = function (cacheUrl, cachePath, uuid) {
+	if (cacheUrl && uuid) {
+		window.pbjs.renderAd(document, "", {
+			cacheHost: cacheUrl,
+			cachePath: cachePath,
+			uuid: uuid,
+			mediaType: "banner",
+		});
+	}
+};
+
+exports.removeProtocolFromUrl = function(url){
+	var outputUrl = url || "";
```

Comment:
Need isString check; if an array is passed as an argument then code might break

Reply:
done